frontend/src/ui/player/Plyr.tsx

```diff
@@ -71,6 +83,45 @@ const PlyrPlayer: React.FC<PlyrPlayerProps> = ({ tracks, title }) => {
         ratio: `${aspectRatio[0]}:${aspectRatio[1]}`,
     };
 
+    // Unfortunately, `plyr-react` does not seem to offer a way to access the
+    // `<video>` element via `ref`. So we just use a unique random ID.
+    const elementId = useId();
+
+    // Setup HLS if we have an HLS track.
+    const hlsRef = useRef<Hls | null>(null);
+    const loadHls = async () => {
+        if (hlsTrack !== undefined) {
+            if (!Hls.isSupported()) {
+                // TODO: improve this. It's fine for now as browsers that don't
+                // support hls.js are very rare by now.
+                throw new Error("HLS is not supported, but required to play this video");
+            }
+
+            const videoElement = document.getElementById(elementId) as HTMLVideoElement;
```

Comment:
This assumes that the id is bound directly to the video element. Is this guaranteed? :thinking:

Reply:
The HLS examples from `plyr-react` do it like this.

Reply:
Oh ok, that's enough for me. :shrug: